core/src/main/java/org/frankframework/http/rest/MediaTypes.java

```diff
@@ -40,7 +41,56 @@ public enum MediaTypes {
 	OCTET("application/octet-stream", null), // raw binary formats do not have a charset
 	MULTIPART_RELATED("multipart/related"),
 	MULTIPART_FORMDATA("multipart/form-data"),
-	MULTIPART("multipart/*");
+	MULTIPART("multipart/*"),
+
+	// less but still common types
+	TXT("text/plain"),
+	CSS("text/css"),
+	CSV("text/csv"),
+	MJS("text/javascript"),
+	HTML("text/html"),
+	JAR("application/java-archive"),
+
+	DOC("application/msword"),
+	EPUB("application/epub+zip"),
+	VSD("application/vnd.visio"),
+	AZW("application/vnd.amazon.ebook"),
+	PPT("application/vnd.ms-powerpoint"),
```

Comment:
Ik denk dat je de openxml gebaseerde varianten ook wilt toevoegen (docx, xlsx, etc)

Reply:
Mocht dat ooit relevant zijn dan kan dat. Maar ik denk niet dat het nodig is.